resources/assets/js/annotations/components/annotationCanvas.vue

```diff
@@ -715,6 +715,21 @@ export default {
                 this.modifyInteraction.setActive(defaultMode);
             }
         },
+        canModify(state){
+            if(!state){
+                this.modifyInteraction.setActive(false);
+            }
```

Comment:
I think you need something like this:

```suggestion
            this.modifyInteraction.setActive(state && this.isDefaultInteractionMode);
```

Otherwise the interaction is not enabled initially.